bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "balance-text",
+  "version": "1.5.0",
+  "authors": [
+    "Adobe Systems",
+    "Inc. <https://github.com/adobe-webplatform>"
```

Comment:
@redmunds whoops, sorry about that!! Looks like `bower init` comma delimits authors, so "Adobe Systems, Inc." was parsed as two authors.

Should be fixed now :+1: